tasks/install.yml

```diff
@@ -37,9 +37,20 @@
     mode: 0755
     owner: "{{ airflow_user }}"
     group: "{{ airflow_user }}"
-  tags: 
+  tags:
     - airflow_setup
 
+- name: Create airflow logs directory structure
+  file:
+    path: "{{ airflow_install_directory }}/logs"
```

Comment:
i would recommend using the variable set from the defaults file `base_log_folder: "{{ airflow_install_directory }}/logs"`

https://github.com/CreditCardsCom/ansible-airflow/blob/master/defaults/main.yml